Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabUndoHelpers.cpp

```diff
@@ -104,6 +105,17 @@ namespace AzToolsFramework
                 state->Redo();
             }
 
+            void UpdateEntitiesAsOverrides(
+                const AZStd::vector<const AZ::Entity*>& entityList,
+                Instance& owningInstance,
+                const Instance& focusedInstance,
+                UndoSystem::URSequencePoint* undoBatch)
+            {
+                PrefabUndoEntityOverrides* state = aznew PrefabUndoEntityOverrides("Undo Updating Entity List As Override");
+                state->SetParent(undoBatch);
+                state->CaptureAndRedo(entityList, owningInstance, focusedInstance);
```

Comment:
nit: maybe add a comment here or in the undo class as to why this undo node does both at the same time as opposed to other nodes.